frontend/tests/controllers/ContestCloneTest.php

```diff
@@ -180,4 +180,37 @@ public function testCreatePrivateContestCloneWithoutAccess() {
             $this->assertSame('userNotAllowed', $e->getMessage());
         }
     }
+
+    /*
+     * Under13 users can't clone contests.
+     */
+    public function testUserUnder13CannotCloneContests() {
+        $contestData = \OmegaUp\Test\Factories\Contest::createContest();
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
+            new \OmegaUp\Test\Factories\UserParams([
+                'birth_date' => strtotime('-10 years'),
+            ]),
+        );
+
+        // Log in the user and set the auth token in the new request
+        $login = self::login($identity);
+
+        try {
+            \OmegaUp\Controllers\Contest::apiClone(
+                new \OmegaUp\Request([
```

Comment:
remember to keep the alignment similar to the rest of the code.

```suggestion
            \OmegaUp\Controllers\Contest::apiClone(new \OmegaUp\Request([
```

same in all other places.

Reply:
> minor formatting comments. we're also missing the test change where the user of exactly 13y _can_ create a contest.

what to do in this case?? could you please example for one

Reply:
https://github.com/omegaup/omegaup/blob/d3bd377aa025a0686f0576067f4cb52a68d941fd/frontend/tests/controllers/ContestCreateTest.php#L13